extensions/converters/list/src/main/java/io/smallrye/config/converter/list/ListConverter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.smallrye.config.converter.list;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.eclipse.microprofile.config.spi.Converter;
+
+/**
+ * Converts a comma separated string to a list
+ * @author <a href="mailto:phillip.kruger@redhat.com">Phillip Kruger</a>
+ */
+public class ListConverter implements Converter<List> {
+
+    @Override
+    public List convert(String input) throws IllegalArgumentException {
+        if(isNullOrEmpty(input))return new ArrayList();
+        
+        Stream<String> stream = Stream.of(input);
+        return stream.collect(Collectors.toList());
```

Comment:
I don't think this would really work right: it just collects a single string to a list with one element.  Normally you'd want to use `io.smallrye.config.SmallRyeConfig#getValues` for this case anyway.

Reply:
This is now removed. Functionality is now part of the spec